src/overlays/actors/ovl_En_An/z_en_an.c

```diff
@@ -2115,6 +2115,7 @@ s32 EnAn_HandleDialogue(EnAn* this, PlayState* play) {
                 case 0x28F5: // "The town will be crushed by the moon. Forget about the letter"
                     this->savedFaceIndex = ENAN_FACE_0;
                     FALLTHROUGH;
+
                 case 0x28A5: // "Visit us again in the future"
```

Comment:
You went the wrong way: Fallthrough's should not have a newline between them

Reply:
Oh, I read it the other way around :sweat_smile: